build/media_source/plg_editors_tinymce/js/plugins/dragdrop/plugin.es6.js

```diff
@@ -168,4 +168,10 @@ window.tinymce.PluginManager.add('jdragndrop', (editor) => {
 
     editor.contentAreaContainer.style.borderWidth = '0';
   });
+  return {
+    getMetadata: () => ({
+      name: 'Drag and Drop (Joomla)',
+      url: 'https://www.joomla.org/'
+    })
```

Comment:
```suggestion
      url: 'https://www.joomla.org/',
    }),
```
Fix javascript-cs failing in drone.